src/docs/sdk/hub_and_scope_refactoring.mdx

```diff
@@ -170,14 +172,84 @@ For more information like the goals (and non-goals) see this GitHub Issue:
 
 https://github.com/getsentry/team-sdks/issues/4
 
+### Isolation Scope
+
+Without an isolation scope, writing to the current `Scope` can lead to unexpected results. Writes can go to a scope that's never applied the way a user would expect. While this is already a problem today, it is exaggerated by moving over to OTel, as OTel auto instrumentation can do lots of context forking, leading to deeply nested scopes. Some hooks are automatically wrapped in a new span by OTel, causing it to also have a separate `Scope`, so the outer `Scope` can't be manipulated.
+
+With isolation scope, we have a place where e.g. hooks can write to that should affect e.g. the whole request.
+
+When isolation scope is forked, the SDK should also fork current scope at the same time. Otherwise users would have to e.g. call both `withScope` and `withIsolationScope` leading to less readable code.
+
 ### Implementation Details
 
-TODO: Add some findings we discovered when implementing this in the first two SDKs.
+#### OTel `Context`
 
-Also add this somewhere: https://miro.com/app/board/uXjVNnI7dsE=/
+OTel stores spans as well as other information in a `Context` and takes care of propagating this `Context` through libraries and threads. We want to rely on this propagation as it allows us to not worry about propagation, instead we just store our scopes in `Context`and trust OTel to propagate it correctly.
 
-### Isolation Scope
+**Can't we simply propagate our `Scopes` ourselves and ignore OTEL Context?**
+If we didn't want to rely on OTels `Context` propagation, we'd have to provide a counterpart for every OTel propagation mechanism. We would no longer be able to just rely on OTEL for instrumentations but have to mirror some of what they do. This would also have to be kept up to date. Whenever OTEL changes something about how they propagate Context we'd have to adapt as well. Keeping `Context` and `Scopes` in sync would be hard and possibly also hard to detect. Version compatibility between Sentry and OTel could also become difficult.
```

Comment:
```suggestion
If we didn't want to rely on OTels `Context` propagation, we'd have to provide a counterpart for every OTel propagation mechanism. We would no longer be able to just rely on OTel for instrumentations but have to mirror some of what they do. This would also have to be kept up to date. Whenever OTel changes something about how they propagate `Context` we'd have to adapt as well. Keeping `Context` and `Scopes` in sync would be hard and possibly also hard to detect. Version compatibility between Sentry and OTel could also become difficult.
```